client/src/entrypoints/draftail.js

```diff
@@ -40,3 +40,33 @@ const plugins = [
 ];
 
 plugins.forEach(draftail.registerPlugin);
+
+class DraftailRichTextArea {
+  constructor(options) {
+      this.options = options;
+  }
+
+  render(container, name, id) {
+    const options = this.options;
+
+    return {
+      getValue() {
+        return input.value;
+      },
+      getState() {
+        return input.value;
+      },
+      setState(state) {
+        // We must initialise it here because Draftail doesn't provide a way to change state after initialisation
+        const input = document.createElement('input');
+        input.type = 'hidden';
+        input.id = id;
+        input.name = name;
+        input.value = state;
+        container.appendChild(input);
+        draftail.initEditor('#' + id, options, document.currentScript);
+      }
+    };
+  }
+}
+window.telepath.register('wagtail.widgets.DraftailRichTextArea', DraftailRichTextArea);
```

Comment:
Not sure it's safe for this to go in draftail.js - draftail as a Django widget is likely to appear on views that don't include telepath.js.